webpack.config.ts

```diff
@@ -45,10 +46,11 @@ const config: ConfigFn = (env, argv) => {
             chemiscope: './src/index.ts',
             'chemiscope-app': './app/app.ts',
             'molecule-viewer': './src/structure/widget.ts',
-        },
+        } as { [key: string]: string },
```

Comment:
```suggestion
        },
```

This should no longer be required